src/jit/codegenxarch.cpp

```diff
@@ -4644,7 +4644,16 @@ void CodeGen::genCodeForIndir(GenTreeIndir* tree)
     else
     {
         genConsumeAddress(addr);
-        emit->emitInsLoadInd(ins_Load(targetType), emitTypeSize(tree), tree->gtRegNum, tree);
+        instruction ins = ins_Load(targetType);
```

Comment:
I don't see any changes to have `ins_Load` return the instructions it doesn't currently know about (non-temporaral, etc).

Reply:
As far as I can tell changing `ins` appropriately is the job of the `if` below.